cfme/tests/middleware/test_middleware_server.py

```diff
@@ -19,11 +21,19 @@
 
 @pytest.yield_fixture(scope="function")
 def server(provider):
-    server = get_server(provider, EAP_PRODUCT_NAME)
+    server = get_server_by_name(provider, EAP_PRODUCT_NAME, 'EAP7')
     yield server
     server.restart_server()
 
 
+@pytest.yield_fixture(scope="function")
+def domain_server(provider):
+    server = get_server_by_name(provider, EAP_PRODUCT_NAME, 'EAP7-server-one')
+    yield server
+    server.start_server()
+    server.restart_server()
```

Comment:
the reason to restart was that 'start' does not help if server is started but suspended, better solution is to resume and then start, it covers all cases and is not so 'hard' as restart.